pkg/virt-handler/vm.go

```diff
@@ -1809,9 +1830,14 @@ func (d *VirtualMachineController) processVmUpdate(origVMI *v1.VirtualMachineIns
 				return err
 			}
 
-			err = d.setPodNetworkPhase1(vmi)
+			criticalNetworkError, err := d.setPodNetworkPhase1(vmi)
```

Comment:
Same as https://github.com/kubevirt/kubevirt/pull/3628/commits/e618e1a7956b1e193b39f012483ddfa18adf1e16#r443652146
this lines 1833-1840 should be located before line 1829.

Reply:
The link is not working.
Why should the order be changed?

Reply:
Same as my comment [commented](https://github.com/kubevirt/kubevirt/pull/3628#discussion_r443988214) on why we should check if there is network connectivity between target and source VM's before starting the migration.

Reply:
The code here is invoked when running a new vm and not during a migration process. However, the same answer I answered there is relevant here as well.